BubbleSort/src/BubbleSort.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Requirements:
+ *
+ * Implement a bubble sort algorithm to sort an array of SalesRepresentative objects in descending order.
+ * The sorting should be done based on the amount of revenue generated via sales.
+ * Each rep has a number of sales, and a quota / sale.
+ *
+ * After the sorting, the first object should be the rep with the most revenue generated.
+ * The last object in the array should be the rep with the least revenue generated.
+ *
+ * E.g
+ *
+ * Sales guy 1 has 10 sales with a quota of 500$. He has achieved 5000$ worth of sales.
+ *
+ * Sales guy 2 has 7 sales with a quota of 800$. He has achieved 5600$ worth of sales.
+ *
+ *
+ * Sorting code example:
+ *
+ * SalesRepresentative [ ] reps = ...
+ *
+ * SalesRepresentative [ ] sortedReps = sort(reps);
+ *
+ * // you need to implement the sort(SalesRepresentative [ ] reps) method. This can not be a static method.
+ */
+
+
+import java.util.Arrays;
+
+public class BubbleSort {
+    public static void main(String[] args) {
+        int salesRepresentative[] = new int[]{1, 2, 3, 4, 5};
+
+        int numberOfSales[] = new int[]{10, 7, 5, 8, 9};
+        int list[] = new int[]{500, 800, 400, 900, 700};
+
+
+        System.out.println("Primul tabel inainte de transformare este: ");
+        for (int n = 0; n < 5; n++) {
+
+            System.out.println(salesRepresentative[n] + " ..... " + numberOfSales[n] + "......" + list[n]);
+        }
+        System.out.println("Lista dupa aplicarea metodei Boubble sort este:" + "");
+
+
+        int bestQuotaSale = getBestQuotaSale(list, salesRepresentative);
+        for (int i = 0; i < list.length; i++) {
+
+        }
+    }
+
+    public static int getBestQuotaSale(int[] list, int[] salesRepresentative) {
+
+        int temp;
+        boolean fixed = false;
+        while (fixed == false) {
+            fixed = true;
```

Comment:
what does fixed mean here? in java the boolean variables are created with isFixed, hasFixedValues... Please rename this and also please add some java doc for this method